### ISSUE TITLE
Collect list of existing resources

### ISSUE THREAD

**Issue by rn-v** [tattle-made/OGBV#30]
For data sampling for annotation, as well as to potentially include them in our training set, we need a list of existing resources for our target languages. Specifically, we need :
- Existing datasets and papers
- Their statistics (label distribution etc.)
- Pre-trained models
- If there's some novelty in how they're modifying their approach for Indic languages

**Comment by rn-v** [comment]
I don't have permissions to assign/label so leaving this as is. @dennyabrain Can you please do that whenever you get a chance?

**Comment by dennyabrain** [comment]
Hey @rn-v, I have sent you an invite for Write access to the repo. that should cover this permission too I think. lmk if it doesn't.

**Comment by rn-v** [comment]
Great, it does, thanks!
